.circleci/config.yml

```diff
@@ -0,0 +1,56 @@
+version: 2.1
+
+orbs:
+  win: circleci/windows@2.4.0
+
+executors:
+  windows-2xlarge:
+    machine:
+      image: 'windows-server-2019-vs2019:stable'
+      resource_class: windows.2xlarge
+      shell: bash.exe
+
+jobs:
+  build:
+    executor: windows-2xlarge
+    
+    environment:
+      THIRDPARTY_HOME: C:/Users/circleci/thirdparty
+      CMAKE_HOME: C:/Users/circleci/thirdparty/cmake-3.16.4-win64-x64
+      CMAKE_BIN: C:/Users/circleci/thirdparty/cmake-3.16.4-win64-x64/bin/cmake.exe
```

Comment:
I asked and got confirmed from CircleCI's forum: https://discuss.circleci.com/t/environment-variable-expansion-in-windows-bash-shell/34933/2 that config.yml does not support variable expansion, so we have to duplicate the `C:/Users/circleci/thirdparty` here. I've created a feature request in https://ideas.circleci.com/ideas/CCI-I-1427